### DISCUSSION TITLE
thirdparty deps for open3d

### DISCUSSION THREAD

**Discussion by youngday** [xmake-io/xmake#4120] [Q&A]
in ubuntu20.04,open3d v0.17.0 is installed, add_requires("cmake::Open3D ",{alias = "open3d", system = true}),
 when i compile examples,find it ,utility and visualization  3rd lib not found , 
errors: Visualizer.cpp:17: undefined reference to `open3d::PrintOpen3DVersion()',
how to add thirdparty of open3d for xmake.lua ?
 Thanks!

**Comment by waruqi** [comment]
add_requires("open3d")
